config/routes.rb

```diff
@@ -391,15 +390,27 @@
       get '/in_person/:step' => 'in_person#show', as: :in_person_step
       put '/in_person/:step' => 'in_person#update'
 
-      get '/by_mail' => 'gpo_verify#index', as: :gpo_verify
-      post '/by_mail' => 'gpo_verify#create'
+      get '/by_mail/enter_code' => 'by_mail/enter_code#index', as: :enter_verify_by_mail_code
+      post '/by_mail/enter_code' => 'by_mail/enter_code#create'
```

Comment:
During the 50/50 state while deploying, a new instance could request /by_mail/enter_code, get an old instance, and 404. I think you have to add the new routes (with redirects to the old ones) in a separate PR before using them.

Reply:
Oof you are right